src/backend/web/routes/admin.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const { UI } = require('bull-board');
+
+const router = express.Router();
+
+router.use('/queues', UI);
```

Comment:
I'm not sure I understand what is going on here. Are you trying to add middleware to the router or the server? If it's the server you're after, the file you should be modifying is app.js. Check out that file and see how the middleware is implemented for the /health endpoint

Reply:
The Bull UI adds a route to an existing express server, yes.  It gives you this dashboard based on queues you are running.